cargo-dist/src/config/v1/mod.rs

```diff
@@ -464,6 +464,22 @@ pub struct TomlLayer {
 }
 
 impl TomlLayer {
+    /// Update `config_version` to the current default.
+    pub fn with_current_config_version(&self) -> Self {
+        let mut new = self.clone();
+        new.config_version = crate::config::ConfigVersion::default();
+        new
+    }
+
+    /// Update `dist_version` to the current dist version.
+    pub fn with_current_dist_version(&self) -> Self {
+        let current_version: semver::Version = std::env!("CARGO_PKG_VERSION").parse()
+            .expect("CARGO_PKG_VERSION was not set -- this is probably a bug in dist; please file an issue!");
+        let mut new = self.clone();
+        new.dist_version = Some(current_version);
+        new
+    }
```

Comment:
These were originally included inline in `migrate/from_v0.rs`, but it made the code a lot harder to understand without any benefit. And, given that it's working with `TomlLayer` itself, it makes more sense to me to include it here than in the migration code.

Reply:
i like this re-org for sure, may be useful to further clarify what current means and what the source of truth for what current is